crates/cheatnet/src/runtime_extensions/call_to_blockifier_runtime_extension/rpc.rs

```diff
@@ -46,6 +46,31 @@ impl UsedResources {
                 .or_insert(0) += count;
         }
     }
+
+    pub(crate) fn subtract_syscall_counter(
+        self: &mut UsedResources,
+        syscall_counter: &SyscallCounter,
+    ) {
```

Comment:
I think it's pretty self explanatory - you subtract syscalls in the `SyscallCounter` you provide as an argument from the syscalls in `UsedResources.execution_resources.syscall_counter`, I don't think we need a docstring here.
When it comes to purpose you can always check the usages, in this case it is to obtain syscalls used in the call and its inner calls (we need to subtract for that since `CallInfo` don't havee `SyscallCounter` yet, so we calculate it ourselves)

Reply:
Changed to a method subtracting `SyscallCounter`s